### PR TITLE
fix(70387): fix table card name --run-build

### PR DESCRIPTION
**Pull request by imam-auliya**
> [#70387] - FE POS - Done

Acceptance Criteria
- [x] Target move table berwarna merah

Changelog:  
- #handle selected table style

image

### CODE REVIEW

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=62) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=62)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=62)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=62)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=62)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=62&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=62)

File: src/app/modules/order/Components/MoveItem.component.tsx [hunk 1]
```diff
@@ -245,6 +245,7 @@ const MoveItemComponent: React.FC<PropMoveItem> = ({
     setupWebsocket(url, payload).onmessage = (event) => {
       const response = checkJson(event.data) ? JSON.parse(event.data) : ''
       if (response?.response_schema?.response_code === '200') {
+        console.log('OUTPUT', response?.response_output)
```
Comment (rse-ait) [inline src/app/modules/order/Components/MoveItem.component.tsx:248]: Please remove unused log

Comment (imam-auliya) [reply src/app/modules/order/Components/MoveItem.component.tsx]: Done mas

**Comment by akarintitech** [comment]
# SonarQube Code Analytics 
## Quality Gate Status: OK

passed

[Sonar report](https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=62) (https://sonar.akarinti.tech/dashboard?id=dayacore_pos_web&pullRequest=62)

## Additional information
*The following metrics might not affect the Quality Gate status but improving them will improve your project code quality.*

## Issues
- Bugs: security_a [0 Bugs](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=62)

- Vulnerabilities: security_a [0 Vulnerabilities](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=VULNERABILITY&pullRequest=62)

- Code Smells: security_a [0 Code Smells](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=62)

- Security Hotspots: security_a [0 Security Hotspots](https://sonar.akarinti.tech/security_hotspots?id=dayacore_pos_web&pullRequest=62&inNewCodePeriod=true)



## Coverage and Duplications
- Coverage: **Coverage**

- Duplications:  [0.00% Duplication](https://sonar.akarinti.tech/project/issues?id=dayacore_pos_web&metric=new_duplicated_lines_density&view=list&pullRequest=62)

**Comment by rse-ait** [comment]
LGTM

**Pull request merged**